Create " AGM Enviroment" for GSOC.md

fernan12345 opened the pull request:
Second post of my project for gsoc

pilarbachiller reviewed (changes requested):
Rename the file of the post with a shorter name and the extension .md. Add also the link to your new post in index.md

fernan12345 commented:
@pilarbachiller Hi pilar, i have already make the changes you asked for, and added all of them in patch-2 branch. Now it is not required to merge the branch1-patch, so i have already removed it.
